es/criteria/open-licenses.md

```diff
@@ -21,7 +21,7 @@ order: 13
 
 * Previene el uso del código por parte de actores específicos.
 
-## Cómo testear
+## Cómo probar
```

Comment:
I would like to keep the word "Testear". It is in the RAE dictionary (https://dle.rae.es/testear?m=form) and also refers directly to the word 'test', as in perform software checks. Perhaps we can use "hacer pruebas o tests"